event_manager/utils.py

```diff
@@ -2,6 +2,12 @@
 import jwt
 from event_manager.settings import SIMPLE_JWT
 
+TEST_USER_CREDENTIALS = {
```

Comment:
Theses should not be in utils, anything test related should be in tests folder @usmandap